pkg/reftracker/ref_key.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package reftracker
+
+type RefKey struct {
+	kind, refName, namespace string
+}
+
+func NewRefKey(kind, refName, namespace string) RefKey {
+	return RefKey{kind, refName, namespace}
+}
+
+func (r RefKey) Kind() string {
```

Comment:
it seems that Kind() is used as part of user messaging. instead let's add Description() method that returns useful description including kind, ref name, ns.

Reply:
Added